recipe/conda_build_config.yaml

```diff
@@ -851,6 +852,13 @@ python_min:
   - '3.9'
 is_freethreading:
   - false
+is_python_min:
+  - true
+  - false
+  - false
+  - false
+is_abi3:
+  - true
```

Comment:
Should `is_abi3` be zipped with `is_freethreading`? Otherwise it seems we would just turn off ABI3 completely when a free-threading build is added